modules/payloads/stagers/osx/x64/reverse_tcp.rb

```diff
@@ -23,9 +23,6 @@ def initialize(info = { })
       'Stager'      => { 'RequiresMidstager' => false }, # Originally set to true, but only Linux payloads use this at the moment, not OSX.
       'Convention'  => 'sockedi',
     ))
-    register_options([
-      OptInt.new('MeterpreterDebugLevel', [ true, "Set debug level for meterpreter 0-3 (Default output is strerr)", 0])
-    ])
```

Comment:
Bit of a weird one here, not needed any longer but it was only added for this one x64 transport type and no others?

Reply:
@dwelch-r7 Do we need to add in logging config to the osx payloads to implement this functionality now? 👀